CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+Changelog
+=========
+Version 0.1.0
+
+				Initial release
```

Comment:
It looks like some hard tabs snuck in here; mind replacing them with 4 space soft tabs?